spec/composite-chart-spec.js

```diff
@@ -327,8 +327,8 @@ describe('dc.compositeChart', function() {
 
                 dateValueGroupLine2.on("click")(dateValueGroupLine2.datum());
                 expect(dateValueGroupLine2.text()).toBe('Date Value Group Line 2');
-                expect(d3.select(chart.selectAll('g.dc-legend g.dc-legend-item')[0][3]).classed("fadeout")).toBeTruthy();                
-                expect(chart.selectAll("path.line").size()).toEqual(3);
+                expect(dateValueGroupLine2.classed("fadeout")).toBeFalsy();
+                expect(chart.selectAll("path.line").size()).toEqual(1);
```

Comment:
the test is looking for a faded out legend item, and it also seems weird for there only to be one line remaining, so i was suspicious. i'm adding a flag for "this is a child chart", which seems like something we need to know anyway.